web/filament-js/jsenums.cpp

```diff
@@ -136,6 +136,10 @@ enum_<View::AntiAliasing>("View$AntiAliasing")
     .value("NONE", View::AntiAliasing::NONE)
     .value("FXAA", View::AntiAliasing::FXAA);
 
+enum_<View::DepthOfFieldOptions::Filter>("View$DepthOfFieldOptions$Filter")
+    .value("NONE", View::DepthOfFieldOptions::Filter::NONE)
+    .value("MEDIAN", View::DepthOfFieldOptions::Filter::MEDIAN);
```

Comment:
I think this will bind MEDIAN to 1 instead of 2.  Can you put a placeholder for 1?

Reply:
I was wondering about that so the right side "View::DepthOfFieldOptions::Filter::MEDIAN" that's not the C++ value?

Reply:
God I'm having a bad day, you're right Mathias, the code is fine as is.